docs/architecture/architecture.md

```diff
@@ -6,48 +6,71 @@ This document describes the architecture of ibc-rs. If you're looking for a high
 
 ![](docs/architecture/assets/repo-context.png)
 
-[What is it core reason developers use Hermes for?]
```

Comment:
Do you think it may be useful to set down some terminology, in particular acronyms? Here are some that may be handy:

IBC = InterBlockchain Communication
    - a distributed protocol, essentially
    - has on-chain + off-chain components

ICS = interchain standards
    - standardization documents capturing the specification of IBC protocol across multiple documents (e.g., spec ICS02 captures the client abstraction of the IBC protocol)


IBC module = on-chain component of IBC
relayer = off-chain component of IBC
Hermes = crate `ibc-relayer-cli` = an implementation of a relayer

Reply:
Yep, I'll add some explanations for these